docs/source/usage/plugins/openPMD.rst

```diff
@@ -116,6 +116,9 @@ PIConGPU command line option          description
 ===================================== ====================================================================================================================================================
 ``--openPMD.period``                  Period after which simulation data should be stored on disk.
 ``--openPMD.source``                  Select data sources and filters to dump. Default is ``species_all,fields_all``, which dumps all fields and particle species.
+``--openPMD.range``                   Define a contiguous range of cells per dimension to dump. Each dimension is separated by a comma.
```

Comment:
Should we add it's in user coordinate system (so a total domain where 0 is start of `CORE + BORDER` on first domain), or is it obvious ?

Reply:
Also, is it total or global domain, and should we clarify it?

Reply:
It is is in global domain coordinates. I assumed it is obvious because the user is giving the domain size via `-d X Y Z` but since you asking we should maybe add it to the docstring.

> Should we add it's in user coordinate system (so a total domain where 0 is start of CORE + BORDER on first domain), or is it obvious ?

The user does not know `CORE + BORDER` this is a developer term. Maybe  we should reverence to the command line parameter `-d`

Reply:
I think where 0 is positioned is probably clear, as we are consistent there with the user side. But total vs. global is not clear imo, as both options theoretically make sense